spec/spec_helper.rb

```diff
@@ -27,7 +27,9 @@
 require 'pathname'
 require 'webmock/rspec'
 require 'gooddata'
-require 'vcr_configurer'
+
+vcr_enabled = !ENV['VCR_ON'] || ENV['VCR_ON'] == 'true' # VCR is enabled by default - set VCR_ON=false to disable
+require 'vcr_configurer' if vcr_enabled
```

Comment:
Minor: We could eliminate this extra `if` by moving the `require` to [the `if` block below](https://github.com/gooddata/gooddata-ruby/pull/1168/files#diff-93830fa29d616f7c87903d08b5b1b29aR70).